config/application.rb

```diff
@@ -25,5 +25,19 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     config.i18n.default_locale = :ru
     config.i18n.available_locales = [:ru, :en]
+
+
+    config.action_mailer.delivery_method = :smtp
+    config.action_mailer.smtp_settings = {
+      authentication: ENV['SMTP_SETTINGS_AUTHENTICATION'],
+      address: ENV['SMTP_SETTINGS_ADDRESS'],
+      port: ENV['SMTP_SETTINGS_PORT'],
+      domain: ENV['SMTP_SETTINGS_DOMAIN'],
+      user_name: ENV['SMTP_SETTINGS_USER_NAME'],
+      password: ENV['SMTP_SETTINGS_PASSWORD']
+    }
+    config.action_mailer.default_options = { from: ENV['DEFAULT_EMAIL'] }
+
+    config.action_mailer.default_url_options = { host: ENV['DEFAULT_URL_OPTIONS_HOST'] }
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.